src/Memory/include/Memory/MemoryInfoListener.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MEMORY_INFO_LISTENER_H_
+#define MEMORY_INFO_LISTENER_H_
+
+#include "capture.pb.h"
+
+namespace orbit_memory {
+
+class MemoryInfoListener {
```

Comment:
The following should be done for all new classes:
Every class should have a brief comment what it is good for/how to use it:
https://google.github.io/styleguide/cppguide.html#Class_Comments

Reply:
Thanks so much for pointing out this. I have added descriptions for all newly added classes in the commit 
*[[fixup] Add class descriptions](https://github.com/google/orbit/pull/1918/commits/5f6e2176ad33d13ac242a42f8e026ce3a8462af3)*.